pkgs/applications/science/chemistry/wxmacmolplt/default.nix

```diff
@@ -7,6 +7,7 @@
 , pkg-config
 , xorg
 , autoreconfHook
+, wrapGAppsHook
```

Comment:
There is only `wrapGAppsHook3` or `wrapGAppsHook4` now.

Reply:
Ah, `wrapGAppsHook` is now merely an alias to `wrapGAppsHook3`. Fixed.